nss-day-cohort-52/rare-api-smoky-pythons
# models

### Issue ticket(s): #48 

### New file(s): post_tag.py, rare_user.py, tag.py

### Modified File(s): models/__init__.py

### Modifications: created tag model, rare_user model, and post_tag model. 

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to lz-models
3. check each model class for the correct variables and variable names.
